scripts/quests/otherAreas/Test_My_Mettle.lua

```diff
@@ -0,0 +1,438 @@
+-----------------------------------
+-- Test My Mettle
+-----------------------------------
+-- Log ID: 4, Quest ID: 25
+-- Devean : !pos -58 -10 6 248
+-- Jar    : !gotoname Jar
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/settings')
+require('scripts/globals/status')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local selbinaID = require('scripts/zones/Selbina/IDs')
+local davoiID = require('scripts/zones/Davoi/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.TEST_MY_METTLE)
+
+-- do not set gil, it is not a static amount
+quest.reward =
+{
+    fame = 30,
+    fameArea = xi.quest.fame_area.SELBINA_RABAO,
+}
+
+local npcEventID =
```

Comment:
While I love the idea of reducing magic numbers, this actually makes things more obtuse.  It's fairly easy to track event processes in the trigger/finish flow, and this forces the next maintainer to look here.

Also, why a subtable with only one entry?  There's no point to this.

Reply:
I saw an opportunity to make the code more readable.  Instead of using short-term memory remembering which values matched, use words.  Is it likely for the IDs to change with subsequent releases of the game?

For the subtable, thought it might be better practice to show that it could be used in situations where there were more than one NPC.